src/Servers/Kestrel/Core/src/Internal/Http2/HPack/StatusCodes.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Net;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2.HPack
+{
+    internal static class StatusCodes
+    {
+        public static string ToStatusBytes(int statusCode)
```

Comment:
It is a copy of the shared StatusCodes.cs logic from except returns a string instead of bytes. I never got around to updating the name. Need to figure out if there is an improvement available to avoid duplication

https://github.com/dotnet/aspnetcore/blob/44230d36a7a6c41d823915b11e83c26447cead7f/src/Shared/runtime/Http2/Hpack/StatusCodes.cs#L81